front/src/modules/timesStops/helpers/scheduleData.ts

```diff
@@ -30,6 +32,9 @@ export function formatScheduleData(scheduleData: ComputedScheduleEntry) {
   return {
     arrival: scheduleData.arrival ? secToHoursString(scheduleData.arrival, true) : '',
     departure: scheduleData.departure ? secToHoursString(scheduleData.departure, true) : '',
-    stopFor: scheduleData.stopFor ? String(scheduleData.stopFor) : '',
+    stopFor:
+      scheduleData.stopFor !== null && scheduleData.stopFor !== undefined
```

Comment:
`stopFor` is `number | null` you shouldn’t need to check for `undefined`

Reply:
You're right. It's done